src/Ledger/Foreign/HSLedger.agda

```diff
@@ -91,56 +97,58 @@ HSP2ScriptStructure .validPlutusScript?    = λ ()
-  { PolicyId                = ℕ
-  ; Value-CommutativeMonoid = +-0-commutativeMonoid
-  ; coin                    = id
-  ; inject                  = id
-  ; policies                = λ x → ∅
-  ; size                    = λ x → 1 -- there is only ada in this token algebra
-  ; property                = λ x → refl
-  ; coinIsMonoidMorphism    = record
-    { mn-homo = record
-      { sm-homo = record { ⟦⟧-cong = λ z → z ; ∙-homo  = λ x y → refl }
-      ; ε-homo  = refl
+  open TA ℕ
+  open TA.TokenAlgebra
+  coinTokenAlgebra : TokenAlgebra
+  Value-CommutativeMonoid coinTokenAlgebra    = +-0-commutativeMonoid
+  coin coinTokenAlgebra                       = id
+  inject coinTokenAlgebra                     = id
+  policies coinTokenAlgebra                   = λ _ → ∅
+  size coinTokenAlgebra                       = λ x → 1 -- there is only ada in this token algebra
+  _≤ᵗ_ coinTokenAlgebra                       = _≤_
+  AssetName coinTokenAlgebra                  = String
+  specialAsset coinTokenAlgebra               = "Ada"
+  property coinTokenAlgebra                   = λ _ → refl
+  coinTokenAlgebra .coinIsMonoidHomomorphism  =
+    record
+      { isMagmaHomomorphism  = record { isRelHomomorphism = record { cong = id } ; homo = λ _ _ → refl }
+      ; ε-homo               = refl
       }
-    }
-  ; _≤ᵗ_                    = _≤_
-  ; DecEq-Value             = record { _≟_ = Data.Nat._≟_ }
-  }
+  DecEq-Value coinTokenAlgebra                = record { _≟_ = Data.Nat._≟_ }
 
 module _ where
   open TransactionStructure
   open PParamsDiff
 
   HSTransactionStructure : TransactionStructure
-  HSTransactionStructure .Ix                      = ℕ
-  HSTransactionStructure .TxId                    = ℕ
-  HSTransactionStructure .epochStructure          = HSEpochStructure
-  HSTransactionStructure .globalConstants         = HSGlobalConstants
-  HSTransactionStructure .AuxiliaryData           = ⊤
-  HSTransactionStructure .crypto                  = HSCrypto
-  HSTransactionStructure .adHashingScheme         = isHashableSet-⊤
-  HSTransactionStructure .ppHashingScheme         = isHashableSelf PParams
-  HSTransactionStructure .ppUpd .UpdateT          = ⊤
-  HSTransactionStructure .ppUpd .updateGroups     = λ _ → ∅
-  HSTransactionStructure .ppUpd .applyUpdate      = λ p _ → p
-  HSTransactionStructure .ppUpd .ppdWellFormed    = λ _ → true
-  HSTransactionStructure .ppUpd .ppdWellFormed⇒WF = λ _ _ x → x
-  HSTransactionStructure .txidBytes               = id
-  HSTransactionStructure .networkId               = tt
-  HSTransactionStructure .tokenAlgebra            = coinTokenAlgebra
-  HSTransactionStructure .DecEq-TxId              = DecEq-ℕ
-  HSTransactionStructure .DecEq-Ix                = DecEq-ℕ
-  HSTransactionStructure .DecEq-Netw              = DecEq-⊤
-  HSTransactionStructure .DecEq-UpdT              = DecEq-⊤
-  HSTransactionStructure .ss                      = HSScriptStructure
+  HSTransactionStructure .Ix              = ℕ
```

Comment:
This is now not properly aligned. Can you restore the old alignment?

Reply:
I've changed it back to the old alignment.  Note that if this file ever gets documented (becomes a literate `lagda` file), it's likely this nice alignment will not be preserved unless we add one more space before all equal signs.  I think we should make the extra space a standard practice, but I won't effect that style/formatting change here (lest the PR reviewers complain about no-op mods 😉).